source/dagger/gameplay/ping_pong/pingpong_playerinput.cpp

```diff
@@ -59,11 +67,50 @@ void PingPongPlayerInputSystem::Run()
     {
         auto &t = view.get<Transform>(entity);
         auto &ctrl = view.get<ControllerMapping>(entity);
+        if (PingPongGameplay::AI)
```

Comment:
this flag should not be inside DAGGER_DEBUG

Reply:
Thanks for the advice Daniil!